2024/opmodes/TwoStickTeleop.java

```diff
@@ -192,6 +218,28 @@ public void handleEvent(RobotEvent e) {
                         // set arm to extend to its highest capacity to lift robot
                         hangingMotor.setTargetPosition(HANGING_FULLY_RETRACTED);
                         break;
+
+                    case BUTTON_X_DOWN:
+                        intakeMotor.setPower(-1);
+                        transportMotor.setPower(1);
+                        // intake pixels into robot
+                        break;
+//                    case BUTTON_X_UP:
+//                        intakeMotor.setPower(0);
+//                        transportMotor.setPower(0);
+//                        // intake pixels into robot
+//                        break;
+                    case LEFT_BUMPER_DOWN:
+                        intakeMotor.setPower(0);
+                        transportMotor.setPower(0);
+                        // intake pixels into robot
```

Comment:
this comment is incorrect, I assume since the power is being set to 0 it should read "stops pixel motor"